tv_apps/browser/js/awesomescreen.js

```diff
@@ -155,18 +156,16 @@ var Awesomescreen = {
       }
     }).bind(this));
 
-    this.bookmark.addEventListener('itemUpdated', (function(e){
-      BookmarkStore.updateCache();
-      this.dialogHidden();
-    }).bind(this));
-
-    this.bookmark.addEventListener('itemRemoved', (function(e){
-      BookmarkStore.updateCache();
+    this.bookmark.addEventListener('hideDialog', (function(e){
```

Comment:
After I look deeper in the ```BookmarkList```, I found that replacing ```itemUpdated``` and ```itemRemoved``` with ```hideDialog``` is some how not logical. When a ```BookmarkList``` is being used by an outside module, the ```BookmarkList``` has no idea that the outside module has a dialog that needs to be hidden. Please stay with the origin design.